src/tests/parser_test.py

```diff
@@ -538,6 +559,29 @@ def test_dunder_all(code):
     assert module.dunder_all == ('foo', 'bar')
 
 
+def test_dunder_all_without_parentheses():
```

Comment:
I'm not sure `without_parenthesis` describes what this is testing. Maybe `single_value`?